src/app/payment/paymentMethod/DigitalRiverPaymentMethod.tsx

```diff
@@ -36,19 +60,7 @@ const DigitalRiverPaymentMethod: FunctionComponent<DigitalRiverPaymentMethodProp
                 usage: 'unscheduled',
                 showTermsOfSaleDisclosure: true,
                 paymentMethodConfiguration: {
-                    disabledPaymentMethods: [
-                        'alipay',
-                        'bPay',
-                        'codJapan',
-                        'klarnaCredit',
-                        'konbini',
-                        'msts',
-                        'payco',
-                        'payPal',
-                        'payPalCredit',
-                        'payPalBilling',
-                        'wireTransfer',
-                    ],
+                    disabledPaymentMethods: disabledPaymentMethodsData,
```

Comment:
Do you have a link to some Digital River documentation which explains this `disabledPaymentMethods` option?

I couldn't find it and am interested how it's meant to be used

Reply:
There isn't in DR documentation portal, but they told me that you can use this site: https://tools.drapi.io/cm/drop-in/drop-in-builder and it's an option where you can enable or disable payment methods, then the page show you the payload.